Cargo.toml

```diff
@@ -16,11 +16,11 @@ proc-macro = true
 [dependencies]
 proc-macro2 = "1.0"
 quote = "1.0"
-syn = { version = "1.0.3", features = ["visit", "extra-traits"] }
+syn = { version = "2.0", features = ["visit", "extra-traits"] }
 
 [dev-dependencies]
 # 1.0.23 requires rustc 1.36, while our msrv is 1.34
-runtime-macros-derive = "0.4.0"
```

Comment:
Commented this and the relevant test out for now, as Cargo tries to build it with syn 2.0 which it's not compatbile with